From 3edd46ef4299389eec328ebeeb3eef4bb85c55ad Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sat, 1 Oct 2022 18:03:17 +0000
Subject: [PATCH] Disable Generated Kernel Headers

...but done in a way so that there's no need to remove generated_kernel_headers from individual projects - thanks @ExpressLuke

---
 build/soong/Android.bp | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index 24ab5a9f..4e35eeb6 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -20,9 +20,6 @@ bootstrap_go_package {
 superior_generator {
     name: "generated_kernel_includes",
 
-    // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
-
     // Directories that can be imported by a cc_* module generated_headers property
     export_include_dirs: [
         "usr/audio/include/uapi",
@@ -32,14 +29,6 @@ superior_generator {
         "usr/techpack/audio/include",
     ],
 
-    // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [
-        "Makefile",
-        "include/**/*",
-        "arch/$(KERNEL_ARCH)/include/**/*",
-        "techpack/audio/include/**/*",
-    ],
 }
 
 cc_library_headers {
-- 
2.34.1

